app/src/main/java/org/jellyfin/mobile/player/PlayerViewModel.kt

```diff
@@ -193,6 +231,10 @@ class PlayerViewModel(application: Application) : AndroidViewModel(application),
         _player.value = null
     }
 
+    private fun getAnalyticsCollector() = DefaultAnalyticsCollector(Clock.DEFAULT).apply {
```

Comment:
Probably a better name to ensure it's not confused with a getter method. You can also move this below the init block/above `setupPlayer()`.
```suggestion
    private fun buildAnalyticsCollector() = DefaultAnalyticsCollector(Clock.DEFAULT).apply {
```

Reply:
Could you also please move this still? :sweat_smile: